DQM/SiPixelMonitorClient/src/SiPixelEDAClient.cc

```diff
@@ -202,7 +203,7 @@ void SiPixelEDAClient::dqmEndLuminosityBlock(DQMStore::IBooker &iBooker,
           nFEDs_ += mefed->getBinContent(i + 1);
       }
     }
-    eSetup.get<SiPixelFedCablingMapRcd>().get(theCablingMap);
+    theCablingMap = *(eSetup.getHandle(cablingMapToken_).product());
```

Comment:
This could be simplified to
```suggestion
    theCablingMap = eSetup.getData(cablingMapToken_);
```
(and probably a comment somewhere explaining that the copy is intentional could help future readers, espectially those who seek to reduce copying)

Reply:
Done in https://github.com/cms-sw/cmssw/pull/32093/commits/41d416db0cf3f01c17266b24b7e31ca5b8a11d46